hail/src/main/scala/is/hail/expr/ir/functions/ReferenceGenomeFunctions.scala

```diff
@@ -85,5 +85,11 @@ class ReferenceGenomeFunctions(rg: ReferenceGenome) extends RegistryFunctions {
           Seq(TString(), TInt32())).get
         If(isValid(Array(contig, pos)), getRef(Array(contig, pos, before, after)), NA(TString()))
     }
+
+    registerRGCode("contigLength", TString(), TInt32(), null) {
+      case (r, rt, (contigT, contig: Code[long])) =>
```

Comment:
what's up with the lowercase long?

Reply:
a bug is up

Reply:
hah, was being interpreted as a type variable.